### ISSUE TITLE
automatic updated to 2.3.1 - Arcade doesn't work

### ISSUE THREAD

**Issue by Nappa5150** [OpenEmu/OpenEmu#4397]
Here is the Console log for OpenEmuHelperApp on iMac 27 MacOS Mojave 10.14.7.
OE Version 2.3 works perfectly, but I can't go back to that version

Process:               OpenEmuHelperApp [2758]
Path:                  /Applications/OpenEmu.app/Contents/MacOS/OpenEmuHelperApp
Identifier:            OpenEmuHelperApp
Version:               2.3.1 (2.3.1)
Code Type:             X86-64 (Native)
Parent Process:        OpenEmu [2755]
Responsible:           OpenEmuHelperApp [2758]
User ID:               501

Date/Time:             2021-01-03 23:50:16.111 +0100
OS Version:            Mac OS X 10.14.6 (18G7016)
Report Version:        12
Anonymous UUID:        06A36838-AC13-EE6C-BA99-1D0C86B56FD0

Sleep/Wake UUID:       64A6D2A3-69C8-410D-AA3C-2C8649E5469C

Time Awake Since Boot: 34000 seconds
Time Since Wake:       12000 seconds

System Integrity Protection: enabled

Crashed Thread:        3  Dispatch queue: com.apple.NSXPCConnection.user.2755

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x4] Symbol missing

Dyld Error Message:
  Symbol not found: _objc_opt_new
  Referenced from: /Users/USER/Library/Application Support/OpenEmu/*/mamearcade_headless.dylib (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libobjc.A.dylib

Thread 0:: Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	0x00007fff799bc21a mach_msg_trap + 10
1   libsystem_kernel.dylib        	0x00007fff799bc768 mach_msg + 60
2   com.apple.CoreFoundation      	0x00007fff4d8689e1 __CFRunLoopServiceMachPort + 327
3   com.apple.CoreFoundation      	0x00007fff4d867f4e __CFRunLoopRun + 1619
4   com.apple.CoreFoundation      	0x00007fff4d8676a9 CFRunLoopRunSpecific + 459
5   com.apple.CoreFoundation      	0x00007fff4d8674b9 CFRunLoopRun + 40
6   org.openemu.OpenEmuKit        	0x0000000104fe1e27 -[OpenEmuXPCHelperAppBase launchApplication] + 287
7   OpenEmuHelperApp              	0x00000001040bea41 main + 76
8   libdyld.dylib                 	0x00007fff798873d5 start + 1

Thread 1:
0   libsystem_pthread.dylib       	0x00007fff79a7a3f0 start_wqthread + 0

Thread 2:
0   libsystem_pthread.dylib       	0x00007fff79a7a3f0 start_wqthread + 0

Thread 3 Crashed:: Dispatch queue: com.apple.NSXPCConnection.user.2755
0   dyld                          	0x0000000105d6f3ba __abort_with_payload + 10
1   dyld                          	0x0000000105d6ebac abort_with_payload_wrapper_internal + 82
2   dyld                          	0x0000000105d6ebde abort_with_payload + 9
3   dyld                          	0x0000000105d2ea9d dyld::halt(char const*) + 343
4   dyld                          	0x0000000105d2ebc7 dyld::fastBindLazySymbol(ImageLoader**, unsigned long) + 167
5   libdyld.dylib                 	0x00007fff7987332e dyld_stub_binder + 282
6   ???                           	0x00000001132a18e0 0 + 4616493280
7   mamearcade_headless.dylib     	0x0000000108e708c0 __13+[OSD shared]_block_invoke + 16
8   libdispatch.dylib             	0x00007fff7983a63d _dispatch_client_callout + 8
9   libdispatch.dylib             	0x00007fff7983bd4b _dispatch_once_callout + 20
10  mamearcade_headless.dylib     	0x0000000108e7089e +[OSD shared] + 46
11  org.openemu.MAME              	0x0000000105344c09 -[MAMEGameCore init] + 169
12  org.openemu.OpenEmuKit        	0x0000000104ff0ba4 -[OpenEmuHelperApp loadWithStartupInfo:error:] + 585
13  org.openemu.OpenEmuKit        	0x0000000104fe2534 -[OpenEmuXPCHelperAppBase loadWithStartupInfo:completionHandler:] + 77
14  com.apple.Foundation          	0x00007fff4fb26157 __NSXPCCONNECTION_IS_CALLING_OUT_TO_EXPORTED_OBJECT_S2__ + 12
15  com.apple.Foundation          	0x00007fff4faf993c -[NSXPCConnection _decodeAndInvokeMessageWithEvent:flags:] + 2528
16  com.apple.Foundation          	0x00007fff4faf8cf3 message_handler + 215
17  libxpc.dylib                  	0x00007fff79abbeec _xpc_connection_call_event_handler + 56
18  libxpc.dylib                  	0x00007fff79ab9e82 _xpc_connection_mach_event + 933
19  libdispatch.dylib             	0x00007fff7983a6dd _dispatch_client_callout4 + 9
20  libdispatch.dylib             	0x00007fff7984f0d6 _dispatch_mach_msg_invoke + 436
21  libdispatch.dylib             	0x00007fff79840792 _dispatch_lane_serial_drain + 268
22  libdispatch.dylib             	0x00007fff7984fc19 _dispatch_mach_invoke + 481
23  libdispatch.dylib             	0x00007fff79840792 _dispatch_lane_serial_drain + 268
24  libdispatch.dylib             	0x00007fff798413c6 _dispatch_lane_invoke + 433
25  libdispatch.dylib             	0x00007fff798496ed _dispatch_workloop_worker_thread + 598
26  libsystem_pthread.dylib       	0x00007fff79a7a611 _pthread_wqthread + 421
27  libsystem_pthread.dylib       	0x00007fff79a7a3fd start_wqthread + 13

Thread 3 crashed with X86 Thread State (64-bit):
  rax: 0x0000000002000209  rbx: 0x0000000000000000  rcx: 0x0000700005602bc8  rdx: 0x0000700005603030
  rdi: 0x0000000000000006  rsi: 0x0000000000000004  rbp: 0x0000700005602c10  rsp: 0x0000700005602bc8
   r8: 0x0000700005602c30   r9: 0x0000000000000000  r10: 0x00000000000000bb  r11: 0x0000000000000246
  r12: 0x00000000000000bb  r13: 0x0000700005603030  r14: 0x0000000000000004  r15: 0x0000000000000006
  rip: 0x0000000105d6f3ba  rfl: 0x0000000000000246  cr2: 0x000000010536f000
  
Logical CPU:     0
Error Code:      0x02000209
Trap Number:     133

**Comment by clobber** [comment]
This is not related to the OpenEmu update. See here for details: https://github.com/OpenEmu/UME-Core/issues/13#issuecomment-753693911

**Comment by chilledbreeze7** [comment]
I have exactly the same problem, I just updated to v2.3.1 and now all of my Arcade games are not working. I tried re-installing v2.3 but now it work  in that version either.
I'm using a Mac Mini (Late 2004) 3Ghz Intel Core i7, 16GB Ram. System: Mojave 10.14.6

Here's some of the Crash report but i've only pasted a bit of it in as its massive:

Process:               OpenEmuHelperApp [782]
Path:                  /Applications/OpenEmu.app/Contents/MacOS/OpenEmuHelperApp
Identifier:            OpenEmuHelperApp
Version:               2.3.1 (2.3.1)
Code Type:             X86-64 (Native)
Parent Process:        OpenEmu [778]
Responsible:           OpenEmuHelperApp [782]
User ID:               501

Date/Time:             2021-01-04 00:25:36.707 +0000
OS Version:            Mac OS X 10.14.6 (18G7016)
Report Version:        12
Anonymous UUID:        37ADE0CE-C98D-9972-C61E-776EFE6C867D


Time Awake Since Boot: 2000 seconds

System Integrity Protection: enabled

Crashed Thread:        2  Dispatch queue: com.apple.NSXPCConnection.user.778

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x4] Symbol missing

Dyld Error Message:
  Symbol not found: _objc_opt_new
  Referenced from: /Users/USER/Library/Application Support/OpenEmu/*/mamearcade_headless.dylib (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libobjc.A.dylib

Thread 0:: Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	0x00007fff7bd8a21a mach_msg_trap + 10
1   libsystem_kernel.dylib        	0x00007fff7bd8a768 mach_msg + 60
2   com.apple.CoreFoundation      	0x00007fff4fcd17b2 __CFRunLoopServiceMachPort + 328
3   com.apple.CoreFoundation      	0x00007fff4fcd0d20 __CFRunLoopRun + 1612
4   com.apple.CoreFoundation      	0x00007fff4fcd0482 CFRunLoopRunSpecific + 455
5   com.apple.CoreFoundation      	0x00007fff4fcd0296 CFRunLoopRun + 40
6   org.openemu.OpenEmuKit        	0x0000000102737e27 -[OpenEmuXPCHelperAppBase launchApplication] + 287
7   OpenEmuHelperApp              	0x0000000101810a41 main + 76
8   libdyld.dylib                 	0x00007fff7bc553d5 start + 1

Thread 1:
0   libsystem_pthread.dylib       	0x00007fff7be483f0 start_wqthread + 0

Thread 2 Crashed:: Dispatch queue: com.apple.NSXPCConnection.user.778
0   dyld                          	0x00000001064853ba __abort_with_payload + 10
1   dyld                          	0x0000000106484bac abort_with_payload_wrapper_internal + 82
2   dyld                          	0x0000000106484bde abort_with_payload + 9
3   dyld                          	0x0000000106444a9d dyld::halt(char const*) + 343
4   dyld                          	0x0000000106444bc7 dyld::fastBindLazySymbol(ImageLoader**, unsigned long) + 167
5   libdyld.dylib                 	0x00007fff7bc4132e dyld_stub_binder + 282
6   ???                           	0x00000001109418e0 0 + 4573108448
7   mamearcade_headless.dylib     	0x00000001065108c0 __13+[OSD shared]_block_invoke + 16
8   libdispatch.dylib             	0x00007fff7bc0863d _dispatch_client_callout + 8
9   libdispatch.dylib             	0x00007fff7bc09d4b _dispatch_once_callout + 20
10  mamearcade_headless.dylib     	0x000000010651089e +[OSD shared] + 46
11  org.openemu.MAME              	0x0000000104c7fc09 -[MAMEGameCore init] + 169
12  org.openemu.OpenEmuKit        	0x0000000102746ba4 -[OpenEmuHelperApp loadWithStartupInfo:error:] + 585
13  org.openemu.OpenEmuKit        	0x0000000102738534 -[OpenEmuXPCHelperAppBase loadWithStartupInfo:completionHandler:] + 77
14  com.apple.Foundation          	0x00007fff51f72157 __NSXPCCONNECTION_IS_CALLING_OUT_TO_EXPORTED_OBJECT_S2__ + 12
15  com.apple.Foundation          	0x00007fff51f4593c -[NSXPCConnection _decodeAndInvokeMessageWithEvent:flags:] + 2528
16  com.apple.Foundation          	0x00007fff51f44cf3 message_handler + 215
17  libxpc.dylib                  	0x00007fff7be89eec _xpc_connection_call_event_handler + 56
18  libxpc.dylib                  	0x00007fff7be87e82 _xpc_connection_mach_event + 933
19  libdispatch.dylib             	0x00007fff7bc086dd _dispatch_client_callout4 + 9
20  libdispatch.dylib             	0x00007fff7bc1d0d6 _dispatch_mach_msg_invoke + 436
21  libdispatch.dylib             	0x00007fff7bc0e792 _dispatch_lane_serial_drain + 268
22  libdispatch.dylib             	0x00007fff7bc1dc19 _dispatch_mach_invoke + 481
23  libdispatch.dylib             	0x00007fff7bc0e792 _dispatch_lane_serial_drain + 268
24  libdispatch.dylib             	0x00007fff7bc0f3c6 _dispatch_lane_invoke + 433
25  libdispatch.dylib             	0x00007fff7bc176ed _dispatch_workloop_worker_thread + 598
26  libsystem_pthread.dylib       	0x00007fff7be48611 _pthread_wqthread + 421
27  libsystem_pthread.dylib       	0x00007fff7be483fd start_wqthread + 13

Thread 3:
0   libsystem_pthread.dylib       	0x00007fff7be483f0 start_wqthread + 0

Thread 4:
0   libsystem_pthread.dylib       	0x00007fff7be483f0 start_wqthread + 0

Thread 2 crashed with X86 Thread State (64-bit):
  rax: 0x0000000002000209  rbx: 0x0000000000000000  rcx: 0x000070000ddf8bc8  rdx: 0x000070000ddf9030
  rdi: 0x0000000000000006  rsi: 0x0000000000000004  rbp: 0x000070000ddf8c10  rsp: 0x000070000ddf8bc8
   r8: 0x000070000ddf8c30   r9: 0x0000000000000000  r10: 0x00000000000000c3  r11: 0x0000000000000246
  r12: 0x00000000000000c3  r13: 0x000070000ddf9030  r14: 0x0000000000000004  r15: 0x0000000000000006
  rip: 0x00000001064853ba  rfl: 0x0000000000000246  cr2: 0x00000001028fd000
  
Logical CPU:     0
Error Code:      0x02000209
Trap Number:     133
